docs/readmes/orc8r/dev_minikube.md

```diff
@@ -105,6 +116,13 @@ helm template orc8r charts/secrets \
   kubectl apply -f -
 ```
 
+Include the following two lines in the `helm template` command to apply the fluentd secrets:
+
+```bash
+--set-file 'secret.certs.files.fluentd\.pem'=${CERTS_DIR}/fluentd.pem \
+--set-file 'secret.certs.files.fluentd\.key'=${CERTS_DIR}/fluentd.key \
+```
+
```

Comment:
If they need to be always added, they should be appended in line 115.
Otherwise the condition under which they should be added must be clearly stated.

Reply:
I moved everything about the fluentd secrets into a separate block, and added a standalone code block instead of referencing where to add lines.